packages/@sanity/preview/src/createPathMaterializer.js

```diff
@@ -36,12 +36,16 @@ export default function createPathMaterializer(observeWithPaths) {
 
       const nextHeads = uniq(missingHeads.map(path => [path[0]]))
 
+      const _isRef = isReference(value)
```

Comment:
Why the underscore? I thought this was pretty much only used to mark private methods/properties in classes and similar?

Reply:
Just me being a scatterbrain. Originally called the variable `_isReference` due to conflict with the function name, then decided to shorten it. Fixed :)